CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
+- Remove `squid` and `pizzly` fusion detection tools [#406](https://github.com/nf-core/rnafusion/pull/406)
```

Comment:
This PR also:
- Changes multiqc template
- change picard module to gatk4

Reply:
Now picard to gatk4 is merged and multiqc template is changed in https://github.com/nf-core/rnafusion/pull/407